src/server/status/server_status.js

```diff
@@ -2,15 +2,24 @@ import _ from 'lodash';
 
 import states from './states';
 import Status from './status';
+import PluginStatus from './plugin_status';
 
 module.exports = class ServerStatus {
   constructor(server) {
     this.server = server;
     this._created = {};
   }
 
-  create(plugin) {
-    return (this._created[plugin.id] = new Status(plugin, this.server));
+  create(id) {
```

Comment:
I don't really care for the "service" distinction personally, perhaps I should remove it from the status page logic. Primary reason being that services are only one thing that could use this new generic status, and since there aren't any "service" specific properties to this I don't think it makes sense to be `createForService()`